Task12/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+    font-family: roboto;
+    src: url("ttf/Roboto-Regular.ttf");
+}
+body{
+    font-family: roboto;
+}
+.shop-price{
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    grid-gap: 12px;
+    grid-auto-flow: dense;
+}
+.shop-price__element{
+    border-radius: 3px;
+    padding: 5px;
+    box-shadow: 0px 2px 5px 0px rgba(191,191,191,1);
+    position: relative;
+}
+.shop-price__element-description{
+    position: absolute;
+    top: 0;
+    left: 0;
+    margin: 5px;
+    background-color: #8caa3c;
+    color: #FFFFFF;
+    opacity: 0;
+
+}
+.shop-price__element:hover{
+    box-shadow:0px 0px 3px 3px rgba(0,0,0,0.49);
+    transition: 0.2s;
+}
+.shop-price__element_big{
+    grid-column: span 2;
+}
+.shop-price__element-image{
+    display: flex;
+    justify-content: center;
+}
+.element-image{
+    min-width: 100%;
+    height: 270px;
+}
+.shop-price__element-block{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 5px 0;
+    font-size: 22px;
+}
+.shop-price__element-block-name{
```

Comment:
I think you can use 
.shop-price__element-block > div:first-child {}
.shop-price__element-block > div:last-child {}

Too much classes